unicodetools/data/ucd/dev/ScriptExtensions.txt

```diff
@@ -21,13 +21,6 @@
 # values in that set is not material, but for stability in presentation
 # it is given here as alphabetical.
 #
-# The Script_Extensions values are presented in sorted order in the file.
-# They are sorted first by the number of Script property values in their sets,
-# and then alphabetically by first differing Script property value.
-#
-# Following each distinct Script_Extensions value is the list of code
-# points associated with that value, listed in code point order.
-#
 # All code points not explicitly listed for Script_Extensions
 # have as their value the corresponding Script property value
```

Comment:
(Or regenerate to pick it up from the other one, but might as well just fix it here.)
```suggestion
# have as their value the corresponding Script property value.
```